src/cpic/solver/heuristic_board_solver.cpp

```diff
@@ -30,25 +30,35 @@ using std::vector;
 
 typedef unsigned short ushort;
 
+// Strategies
 inline void lookForOneColorLeftColumns(const Board *, BoardState *);
-inline void paintBlanksOnColumn(const Board *, BoardState *, ushort column, Color);
 inline void lookForOneColorLeftRows(const Board *, BoardState *);
+inline void completeContiguousRows(const Board *, BoardState *);
+inline void doTheThing(const Board *, BoardState *);
+
+// Tools
+inline void paintBlanksOnColumn(const Board *, BoardState *, ushort column, Color);
 inline void paintBlanksOnRow(const Board *, BoardState *, ushort row, Color);
```

Comment:
There should a FIXME to remove the `Board*` parameter from these `paintBlank` functions